scripts/src/main/resources/scripts/command/rewrite

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doRun() {
+    local recipe="${1}"
+    local tool="${2}"
+
+    doRunCommand "mvn org.openrewrite.maven:rewrite-maven-plugin:4.39.0:run \
```

Comment:
Optionally you can always pick up the latest version through
```suggestion
    doRunCommand "mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
```

Reply:
@timtebeek awesome. Thanks for your constructive hint. I am very happy to get unexpected feedback to ease my review work :)